modules/components/pages/redpanda/about.adoc

```diff
@@ -2,16 +2,43 @@
 
 component_type_dropdown::[]
 
-As well as the xref:components:logger/about.adoc[default logger], you can configure Redpanda Connect to send logs to a topic in a Redpanda cluster.
+// tag::single-source[]
 
-The configuration for this feature lives under the `redpanda` namespace, with the following default values:
+// © 2024 Redpanda Data Inc.
 
-[source,yaml]
-----
+As well as the xref:components:logger/about.adoc[default logger], you can configure Redpanda Connect to send logs and status updates to topics on a Redpanda cluster.
```

Comment:
Could we give an example of the status updates? For example, would this be when a pipeline moves from started to running? What information will customers be able to see?:

Reply:
We have a proto spec that's probably worth linking to as it describes the full schema of a status update: https://github.com/redpanda-data/connect/blob/main/internal/protoconnect/status.pb.go

Reply:
@Jeffail. I have added this to the `status_topic` definition. Is that OK?

Reply:
Yeah that's fine with me